src/dataset-builder/ConceptSearch.ts

```diff
@@ -109,11 +109,11 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
               paddingBottom: 10,
             },
             columns: [
-              { header: strong(['Concept name']), width: 710, key: 'name' },
-              { header: strong(['Concept ID']), width: 195, key: 'id' },
-              { header: strong(['Code']), width: 195, key: 'code' },
-              { header: strong(['Number of Participants']), width: 205, key: 'count' },
-              { width: 100, key: 'hierarchy' },
+              { header: strong(['Concept name']), key: 'name', size: { grow: 2.2 } },
+              { header: strong(['Concept ID']), key: 'id', size: { grow: 1 } },
```

Comment:
I think we talked about maybe dropping the size of `Concept ID` and `Code` a bit? Maybe .5 for those as well.

Reply:
Some concept codes are pretty long, especially in observation and device, so I'm opting for 0.75 for Code and Number of Participants since I was running into some cases where they would overlap each other a bit - I'll push that change in a second

Reply:
I added a screenshot with the smaller comments, please take a look and let me know what you think when you get the chance!